bin/check-s3-object.rb

```diff
@@ -158,12 +158,16 @@ def run
       elsif !config[:key_prefix].nil?
         key_search = config[:key_prefix]
         output = s3.list_objects(bucket: config[:bucket_name], prefix: key_search)
-        key_fullname = output.contents[0].key
+
+        if output.contents.size.to_i < 1
```

Comment:
Just curious: was there a reason to check `< 1` rather than `== 0` or `.zero?` the only reason I can come up with is trying to account for negative numbers.

Reply:
Honestly? It was some ridiculous hour and it didn't even occur to me to use `== 0` (and I didn't know about `.zero?`) :joy:

Reply:
I know the feeling all too well.

Reply:
If you don't mind let's open up a PR to fix that so that I am not later trying to remember what edge case that covered.